### ISSUE TITLE
Improve Scan Status Reporting for Screen Readers

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#101]
Blind screen reader users have no way of understanding when a scan is complete because scan results are dynamically populated.

**Comment by TomerPacific** [comment]
@bbertucc , I'd like to help out.
What exactly would you like done in this feature?

**Comment by bbertucc** [comment]
@TomerPacific - Adding an `aria-live` tag to the scanning terminal should be the way to go. 

Here is the area of code where the scan status is reported:
https://github.com/bbertucc/equalify/blob/dbe6db045f2a09e39629eb3d77e95083fde0c621/views/scan.php#L20-L22

Once that tag is applied, folks should test the scan with various screen readers.

The goal is for the screen reader to report that the `#terminal` is a live region and read out the status of the scan as it loads.

**Comment by TomerPacific** [comment]
Thanks for the clarification , @bbertucc .
I tried looking for instructions on how to set up a local environment so I can test this out.

I noticed the **"Download and Use"** section of the README, but I don't know which details I need to put it inside `config.php` and how to run a Linux server.

Could you point me in the right direction? 

Thanks

I also read about the [aria-live tag](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions) and it can have three possible values:

1. The default value which is off
2. polite
3. assertive

Each of it differs in it's use case, but it seems that polite is the most common use case.

**Comment by TomerPacific** [comment]
@bbertucc ?

**Comment by bbertucc** [comment]
> @bbertucc ?

@TomerPacific - I started a new issue (#106) with expanded installation instructions. You can jump on that thread with any installation-related questions.

**Comment by TomerPacific** [comment]
Thanks @bbertucc . I'll be sure to go over it and post any follow up questions that I have there.
Regarding the aria-live tag, do you want me to proceed with it and you will review things in the PR?

**Comment by bbertucc** [comment]
> Regarding the aria-live tag, do you want me to proceed with it and you will review things in the PR?

Please test it on a screen reader before you submit. I'm not 100% that the `aria-live` will resolve this ticket.

**Comment by TomerPacific** [comment]
@bbertucc , I'm not certain if I'll be able to have a local environment up and running to test the changes out (I signed up for the Equalify service.
Since I do not know how long this will take, if you prefer someone else help out with this issue, go ahead.
